frontend/ui/uimanager.lua

```diff
@@ -1570,6 +1570,15 @@ function UIManager:powerOff()
     end
 end
 
+function UIManager:restart(message_text)
```

Comment:
The method might better be named `:adviseRestartNeeded(message_text)` (or a better name) - as it always shows `This will take effect on next restart`.
(as we might need to have someday a `UIManager:restart() UIManager:broadcastEvent(Event:new("Restart")) end` that does it directly for some reason without that message)

Reply:
Why not. @NiLuJe will have the final word about namings in UIManager.

Not sure how these other are used, and if there should be a difference between explicite reboot/poweroff requested vs advised restart - or may be your :restart() (and the others) should just have another parameter `(advise_msg=true for the default text, or some alt ext)`, and if `advise_msg` not nil/false, do the ConfirmBox, otherwise proceed directly ?

Reply:
A simple rename sounds good to me, FWIW.

Reply:
Rename of what to what?
And if all renamed to askFor... what about the need for a direct reboot/restart?
And what about multipurpose single methods with params?